WcaOnRails/app/controllers/registrations_controller.rb

```diff
@@ -141,8 +141,14 @@ def update_all
 
   def update
     @registration = Registration.find(params[:id])
-    was_accepted = @registration.accepted?
-    if @registration.update_attributes(registration_params)
+    @competition = @registration.competition
+    if params[:from_admin_view] && @registration.updated_at.to_datetime != params[:registration][:updated_at].to_datetime
```

Comment:
We were worried about the case where a competitor changes their registration *while* an admin is looking at that registration, and then the admin desides to approve their registration. I was concerned this would create a window where registrants could potentially sign up for events unbeknownst to the competition organizers.

However, on further thought, last-write-wins here means that if the admin were to approve the registration in the above example, they'd probably actually end up cloberring all the fields of the user's registration, not just the pending/approved status. So, I agree, it's best to just remove this and the corresponding tests. Sorry for leading you down a bad path, @flee135!